README.md

```diff
@@ -171,3 +171,14 @@ If found, it indicates that the new package version exists in PyPI.
 7. Ensure the release notes contain an upgrade path for any breaking changes
 8. Check PyPI for the new published version
 
+
+## Docker Images
+
+### Building the Images
+
+If building on a ARM mac, the image will build but will fail to deploy with the following error:
+exec /usr/bin/dumb-init: exec format error
+
+Instead, build the image via the below command, to build for the linux/amd64 platform.
+
+`DOCKER_DEFAULT_PLATFORM=linux/amd64 docker build --tag public.ecr.aws/uktrade/nginx-dbt-platform:<tag> .`
```

Comment:
I think either move this stuff to the [nginx readme ](https://github.com/uktrade/platform-tools/blob/ddf8085f6385a033c0235b6d4f80a2c23258ed2f/images/nginx-dbt-platform/README.md) or make the tag non-specific to nginx

Reply:
Will move the comment over